packages/@sanity/default-layout/src/components/RenderTool.js

```diff
@@ -1,28 +1,101 @@
+import React, {Component} from 'react'
 import PropTypes from 'prop-types'
-import React from 'react'
 import tools from 'all:part:@sanity/base/tool'
+import Button from 'part:@sanity/components/buttons/default'
+import ErrorBorkImage from './ErrorBorkImage'
+import styles from './RenderTool.css'
+
+class RenderTool extends Component {
```

Comment:
This is a matter of preference, but IMO, exporting immediately (`export default class RenderTool`) is preferable since the reader don't have to scroll all the way to the bottom of the file to figure out if there's any wrappers or HOCs involved.